apps/channels/views.py

```diff
@@ -27,6 +27,20 @@ def new_whatsapp_message(request):
     return HttpResponse()
 
 
+@csrf_exempt
+def new_turn_message(request, experiment_id: uuid):
+    message_data = json.loads(request.body.decode("utf-8"))
+    if "statuses" in message_data:
+        # Ignore status updates
+        return HttpResponse()
+
+    try:
+        tasks.handle_turn_message.delay(experiment_id=experiment_id, message_data=message_data)
+    except Exception as e:
```

Comment:
Is the error handling necessary?

Reply:
No. It was used for testing. I'll remove